src/contact.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Contact</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <form id="contactForm" action="/submit_contact" method="post">
+    <label for="name">Name:</label><br>
```

Comment:
After the styles are applied, the <br> tags will no longer be needed, so we can just removed them now